src/domain/model/user/AuthenticationValue.ts

```diff
@@ -0,0 +1,3 @@
+export class AuthenticationValue {
```

Comment:
Thought we changed this to Identifier?  Or is that not merged yet?

Reply:
Good catch, I think it's a leftover from the branch before the changes indeed, will remove 👍